substrate/frame/support/procedural/src/lib.rs

```diff
@@ -1122,164 +910,60 @@ pub fn call(_: TokenStream, _: TokenStream) -> TokenStream {
 ///
 /// ---
 ///
-/// **Rust-Analyzer users**: See the documentation of the Rust item in
-/// [`frame_support::pallet_macros::call`](../../frame_support/pallet_macros/attr.call.html).
+/// Rust-Analyzer Users: Documentation for this macro can be found at
+/// `frame_support::pallet_macros::call_index`.
 #[proc_macro_attribute]
 pub fn call_index(_: TokenStream, _: TokenStream) -> TokenStream {
 	pallet_macro_stub()
 }
 
-/// Each dispatchable may be annotated with the `#[pallet::feeless_if($closure)]` attribute,
-/// which explicitly defines the condition for the dispatchable to be feeless.
-///
-/// The arguments for the closure must be the referenced arguments of the dispatchable function.
-///
-/// The closure must return `bool`.
 ///
-/// ### Example
-/// ```ignore
-/// #[pallet::feeless_if(|_origin: &OriginFor<T>, something: &u32| -> bool {
-/// 		*something == 0
-/// 	})]
-/// pub fn do_something(origin: OriginFor<T>, something: u32) -> DispatchResult {
-///     ....
-/// }
-/// ```
-///
-/// Please note that this only works for signed dispatchables and requires a signed extension
-/// such as `SkipCheckIfFeeless` as defined in `pallet-skip-feeless-payment` to wrap the existing
-/// payment extension. Else, this is completely ignored and the dispatchable is still charged.
+/// ---
 ///
-/// ### Macro expansion
+/// Rust-Analyzer Users: Documentation for this macro can be found at
 ///
-/// The macro implements the `CheckIfFeeless` trait on the dispatchable and calls the corresponding
-/// closure in the implementation.
+/// `frame_support::pallet_macros::feeless_if`.
 #[proc_macro_attribute]
 pub fn feeless_if(_: TokenStream, _: TokenStream) -> TokenStream {
 	pallet_macro_stub()
 }
 
-/// Allows you to define some extra constants to be added into constant metadata.
 ///
-/// Item must be defined as:
-///
-/// ```ignore
-/// #[pallet::extra_constants]
-/// impl<T: Config> Pallet<T> where $optional_where_clause {
-/// 	/// $some_doc
-/// 	$vis fn $fn_name() -> $some_return_type {
-/// 		...
-/// 	}
-/// 	...
-/// }
-/// ```
-/// I.e. a regular rust `impl` block with some optional where clause and functions with 0 args,
-/// 0 generics, and some return type.
+/// ---
 ///
-/// ## Macro expansion
+/// Rust-Analyzer Users: Documentation for this macro can be found at
 ///
-/// The macro add some extra constants to pallet constant metadata.
+/// `frame_support::pallet_macros::extra_constants`.
```

Comment:
If anything, an idealistic way to do what I suggested elsewhere is add a feature to `docify` that can "mirror" the docs. Docify is already parsing the rust code and building the syntax tree, and should easily be able to do this. But, I think we will end up with a messy python/bash for now :p